web/package.json

```diff
@@ -3,12 +3,14 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "dev": "concurrently -n NEXT,TS -c magenta,cyan \"next dev -p 3000\" \"yarn ts --watch\"",
-    "devdev": "cross-env NEXT_PUBLIC_FIREBASE_ENV=DEV concurrently -n NEXT,TS -c magenta,cyan \"cross-env FIREBASE_ENV=DEV next dev -p 3000\" \"cross-env FIREBASE_ENV=DEV yarn ts --watch\"",
```

Comment:
@akrolsmir do you know why these old `FIREBASE_ENV=DEV` assignments were in here? They seem useless to me.

Reply:
no, I gave up on understanding that command entirely and just have been cargo-culting it ever since